chain/icon/sender.go

```diff
@@ -550,10 +551,10 @@ func (s *sender) sendFragmentations(tps []*TransactionParam, idxs []int) (chain.
 		if err != nil {
 			s.l.Panicf("sendFragmentations: fails result %+v error%+v", grp, err)
 		}
-		// TODO use (finalizelatency * blockinterval / appropriateratio) * time.Second
-		time.Sleep(time.Duration(s.FinalizeLatency()) * time.Second)
 	}
 
+	// Only last fragmnetation need this
+	time.Sleep(time.Duration(s.FinalizeLatency()*s.iconBlockInterval) * time.Second)
```

Comment:
@trmaphi Could you explain why should we sleep here?

Reply:
This sleep is related what I explained above. It is not enough to resend the TX which is skipped. we talked in Slack that we should throttle TX by confirming the previous TX is in the EXECUTION state. So we need to check TX result while we send new TX.

Reply:
@dhlee-icon i know, that's why i asked him. It's a different point.

Reply:
If you think that it is needed to check tx state when it is sent, it is very okay

Reply:
This sleep is to ensure that the next transaction after the last fragmentation transaction is submitted on the next block. To prevent too many execution time in the same block as the last fragmentation transaction.

The more accurate throttling is make a polling to get transaction status until two JSON errors `JsonrpcErrorCodePending, JsonrpcErrorCodeExecuting` doesn't appear anymore.

Reply:
@trmaphi As far as I understand, It is okay to send tx when the previous one is in the state of Executing
(JsonrpcErrorCodeExecuting) whereas not okay to send tx when the previous one is in the state of Pending(JsonrpcErrorCodePending)

Reply:
@dhlee-icon, state like PENDING (JsonrpcErrorCodePending) → EXECUTING (JsonrpcErrorCodeExecuting ) →SKIPPED still can happen, please check slack

Reply:
Hmm, I don't think so. Once a transaction has returned its result. It means it already have both states: JsonrpcErrorCodePending, JsonrpcErrorCodeExecuting.

Reply:
```
MoonKyu Song - ICONLOOP  3 days ago
PENDING → EXECUTING → SKIPPED is official flow.
Any of the states can be skipped except the final.
```

Reply:
To confirm the transaction is not skipped, we have to wait for both `PENDING` and `EXECUTING` happened, and get the final result